2021-02-01 16:06:55,085 INFO    MainThread:6245 [wandb_setup.py:_flush():70] setting env: {}
2021-02-01 16:06:55,086 INFO    MainThread:6245 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-02-01 16:06:55,086 INFO    MainThread:6245 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-01 16:06:55,086 INFO    MainThread:6245 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-01 16:06:55,086 INFO    MainThread:6245 [wandb_init.py:_log_setup():318] Logging user logs to /home/iiitd/Desktop/Ahmad/doa2/wandb/run-20210201_160655-1siquj0b/logs/debug.log
2021-02-01 16:06:55,086 INFO    MainThread:6245 [wandb_init.py:_log_setup():319] Logging internal logs to /home/iiitd/Desktop/Ahmad/doa2/wandb/run-20210201_160655-1siquj0b/logs/debug-internal.log
2021-02-01 16:06:57,932 INFO    MainThread:6245 [wandb_run.py:_console_start():1308] atexit reg
2021-02-01 16:06:57,933 INFO    MainThread:6245 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.REDIRECT
2021-02-01 16:06:57,933 INFO    MainThread:6245 [wandb_run.py:_redirect():1178] Redirecting console.
2021-02-01 16:06:57,933 INFO    MainThread:6245 [redirect.py:install():205] install start
2021-02-01 16:06:57,933 INFO    MainThread:6245 [redirect.py:install():220] install stop
2021-02-01 16:06:57,933 INFO    MainThread:6245 [redirect.py:install():205] install start
2021-02-01 16:06:57,933 INFO    MainThread:6245 [redirect.py:install():220] install stop
2021-02-01 16:06:57,933 INFO    MainThread:6245 [wandb_run.py:_redirect():1222] Redirects installed.
2021-02-01 16:09:49,076 INFO    MainThread:6245 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 1
2021-02-01 16:09:49,077 INFO    MainThread:6245 [wandb_run.py:_restore():1250] restore
2021-02-01 16:09:49,077 INFO    MainThread:6245 [redirect.py:uninstall():224] uninstall start
2021-02-01 16:09:49,077 INFO    MainThread:6245 [redirect.py:_stop():279] _stop: stdout
2021-02-01 16:09:49,077 INFO    stdout    :6245 [redirect.py:_pipe_relay():121] relay done saw last write: stdout
2021-02-01 16:09:49,077 INFO    stdout    :6245 [redirect.py:_pipe_relay():137] relay done done: stdout
2021-02-01 16:09:49,077 INFO    MainThread:6245 [redirect.py:_stop():285] _stop closed: stdout
2021-02-01 16:09:49,078 INFO    MainThread:6245 [redirect.py:_stop():291] _stop joined: stdout
2021-02-01 16:09:49,078 INFO    MainThread:6245 [redirect.py:_stop():293] _stop rd closed: stdout
2021-02-01 16:09:49,078 INFO    MainThread:6245 [redirect.py:uninstall():228] uninstall done
2021-02-01 16:09:49,078 INFO    MainThread:6245 [redirect.py:uninstall():224] uninstall start
2021-02-01 16:09:49,078 INFO    MainThread:6245 [redirect.py:_stop():279] _stop: stderr
2021-02-01 16:09:49,078 INFO    MainThread:6245 [redirect.py:_stop():285] _stop closed: stderr
2021-02-01 16:09:49,078 INFO    stderr    :6245 [redirect.py:_pipe_relay():121] relay done saw last write: stderr
2021-02-01 16:09:49,078 INFO    stderr    :6245 [redirect.py:_pipe_relay():137] relay done done: stderr
2021-02-01 16:09:49,078 INFO    MainThread:6245 [redirect.py:_stop():291] _stop joined: stderr
2021-02-01 16:09:49,078 INFO    MainThread:6245 [redirect.py:_stop():293] _stop rd closed: stderr
2021-02-01 16:09:49,078 INFO    MainThread:6245 [redirect.py:uninstall():228] uninstall done
2021-02-01 16:09:49,734 INFO    MainThread:6245 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "53a2331c39434d929d233c1ac6c98266"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 511
      total_bytes: 511
    }
  }
}

2021-02-01 16:09:51,739 INFO    MainThread:6245 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "8833eaeafea04c9cbe9005134b8307ef"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 5641
      total_bytes: 5641
    }
  }
}

2021-02-01 16:09:53,018 INFO    MainThread:6245 [wandb_run.py:_show_files():1614] logging synced files
2021-02-01 16:09:53,020 INFO    MainThread:6245 [internal.py:handle_exit():139] Internal process exited
